orchestrator/api/shopify.py

```
@@ -566,25 +383,15 @@ async def forward_event(
-                    "[PRD-009] Scheduled incremental graph update for workspace=%s reason=%s",
-                    ws.id, request.event,
-                )
-            except Exception as e:
-                logger.warning(
-                    "[PRD-009] Could not schedule incremental update: %s", e
-                )
+            import asyncio as _asyncio
+
+            _asyncio.create_task(
+                _sync_catalog_for_workspace(str(ws.id), request.event)
+            )
```

Ruff (0.15.20)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)